app/github_public_repo/README.md

```diff
@@ -0,0 +1,20 @@
+
+# SARS-CoV-2 Serosurveillance Data by Serotracker
+
+SeroTracker synthesizes findings from thousands of COVID-19 seroprevalence studies worldwide, providing a data platform and interactive dashboard for SARS-CoV-2 serosurveillance. This dataset represents our collection of serosurveillance studies
+
+### Download
+Our complete SARS-Cov-2 dataset is available as a CSV programmatically with the following command:
```

Comment:
Change:

Our complete SARS-Cov-2 dataset can be scraped programmatically from this repository in the following ways:

1) Using the command: `curl https://raw.githubusercontent.com/serotracker/sars-cov-2-data/main/test.csv`
2) Using a GET request through REST API protocol